compiler/src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -36,7 +36,7 @@ class ScalaSettings extends Settings.SettingGroup {
   val color: Setting[String] = ChoiceSetting("-color", "mode", "Colored output", List("always", "never"/*, "auto"*/), "always"/* "auto"*/) withAbbreviation "--color"
   val source: Setting[String] = ChoiceSetting("-source", "source version", "source version", List("3.0", "3.1", "3.0-migration", "3.1-migration"), "3.0").withAbbreviation("--source")
   val target: Setting[String] = ChoiceSetting("-target", "target", "Target platform for object files. All JVM 1.5 targets are deprecated.",
-    List("jvm-1.5", "jvm-1.5-fjbg", "jvm-1.5-asm", "jvm-1.6", "jvm-1.7", "jvm-1.8", "msil"), "jvm-1.8") withAbbreviation "--target"
+    List("jvm-1.5", "jvm-1.5-fjbg", "jvm-1.5-asm", "jvm-1.6", "jvm-1.7", "jvm-1.8", "jvm-9", "msil"), "jvm-1.8") withAbbreviation "--target"
```

Comment:
While you're at it, you could also remove all the options except jvm-1.8 and jvm-9 since they never worked in dotty (there's even `msil` which is supposed to generate .NET bytecode!)